SubEthaEdit-Mac/Source/SEEFindAndReplaceState.m

```diff
@@ -276,11 +276,11 @@ - (void)takeValuesFromDictionaryRepresentation:(NSDictionary *)aDictionaryRepres
 
 - (NSString *)menuTitleDescription {
 	NSMutableArray *components = [NSMutableArray new];
-	[components addObject:[NSString stringWithFormat:@"›%@‹", self.findString]];
```

Comment:
Don't know f I see the removal of the angle braces an improvement.

Reply:
Fair enough! It confused me when I first saw it in that menu, I thought it was a glitch. I've never seen these in use before. 

I looked around a bit and found out other apps aren't using any braces in their recents. Which doesn't mean SEE should do the same, of course 😉

Reply:
I do agree all of this is quite cryptic though. If you like it clearly better this way I might get convinced as less often is more.